lib/msal-node/src/cache/CacheManager.ts

```diff
@@ -1,8 +1,15 @@
+/*
```

Comment:
I've split serialize/deserialize from writeToPersistence and readToPersistance, as that seemed much more transparent to me, but I'm happy to discuss/ change it back if want to do that.